### DISCUSSION TITLE
Can be used without MVVM on android?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaInside/Shell#51] [Q&A]
Can be used on android without mvvm? there is any sample of that?

**Comment by OmidID** [comment]
Hello @AMT-Cheif 
Yes. Can be, Please take a looks to the sample app in the repo. 
Shell is an UI component and has nothing with application architecture.

**Comment by OmidID** [comment]
@AMT-Cheif Do you still need help?
